Server/VikingVault.DataAccess/Migrations/VikingVaultDbContextModelSnapshot.cs

```diff
@@ -113,13 +147,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("PictureLink");
 
-                    b.Property<string>("Role");
+                    b.Property<int?>("RoleId");
```

Comment:
Should this roleId be nullable?

Reply:
Yes, it should, but we decided to avoid sending the role from the frontend and pick the role from the backend accordingly. As a consequence, we needed to set this field on not required in order to be able to use this model hence the nullable property. In practice, this field will never be null because it will be automatically completed on the backend when a certain user is created -- regular user or company in this case.

Reply:
If it will never be null, then it shouldn`t be nullable :).